static/templates/task/base.html

```diff
@@ -2,7 +2,7 @@
   <div class="_main-content-body">
     <div class="md-whiteframe-z1 md-padding" >
       <h3>Task #{{ task.taskData.id }} ({{ task.taskData.project_data.name }})</h3>
-
+      <div timer-directive my-text ='task.time_left'></div>
```

Comment:
Can you change it to better names like 
a) timer-counter instead of timer-directive
b) seconds-left instead of my-text

Please validate value passed in my-text